Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/DriverSSNExtra.hs

```diff
@@ -29,3 +31,16 @@ upsert driverSsn@(Domain.Types.DriverSSN.DriverSSN {..}) = do
         ]
         [Se.And [Se.Is Beam.driverId $ Se.Eq (Kernel.Types.Id.getId driverId)]]
     else createWithKV driverSsn
+
+findBySSN :: (EsqDBFlow m r, MonadFlow m, CacheFlow m r, EncFlow m r) => (Kernel.External.Encryption.DbHash -> m (Maybe Domain.Types.DriverSSN.DriverSSN))
+findBySSN ssn = do findOneWithKV [Se.Is Beam.ssnHash $ Se.Eq ssn]
```

Comment:
Why didn't we use DSL :(

Reply:
Because in DSL we have it as 
`ssn: EncryptedHashedField e Text`
But I only wanted to compare it on hash.